openassessment/templates/openassessmentblock/staff/oa_staff_complete.html

```diff
@@ -0,0 +1,22 @@
+{% load i18n %}
+
+<li id="openassessment__staff-assessment" class="openassessment__steps__step step--staff-assessment is--empty is--collapsed is--complete">
```

Comment:
The combination of is--empty and is--collapsed controls the size of the step box, and prevents the box "growing" on hover (which would indicate that it can be expanded). I copied the class structure of other similar templates (https://github.com/edx/edx-ora2/blob/master/openassessment/templates/openassessmentblock/self/oa_self_complete.html#L5).

That being said, it is interesting that some of the steps (like cancelled) don't have these classes, and instead have a slightly larger box (with no hover action). I don't quite the subtleties of the styling, but I took care to make this step have the same styling as the surrounding steps.